specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -819,6 +867,27 @@
           "type": "string"
         }
       }
+    },
+    "AcsCallingCommunicationIdentifierProperties": {
+      "description": "Schema for Calling communication identifier",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../communication/data-plane/Common/stable/2022-07-13/common.json#/definitions/CommunicationIdentifierModel"
+        }
+      ],
+      "properties": {
+        "unknownIdentifiers": {
```

Comment:
I don't get what this is for, the CommunicationIdentifierModel already allows you to express an unknown identifier like this:
```json
{
  "kind": "unknown",
  "rawId": "12345"
}
```

Can we just remove `AcsCallingCommunicationIdentifierProperties` model and use `CommunicationIdentifierModel` directly?

Reply:
Yes, you need to change the implementation of what you send to conform with how identifiers work across ACS APIs. Thanks for updating!